docker/docker-compose.yaml

```diff
@@ -7,6 +7,7 @@ services:
     image: unstract/backend:${VERSION}
     container_name: unstract-backend
     restart: unless-stopped
+    entrypoint: ./entrypoint.sh
```

Comment:
@chandrasekharan-zipstack Entrypoint is set in the base image itself, hence we don't need to set it explicitly here.
Please revert.